devfiles/cpp/devfile.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: cpp-
+projects:
+  -
+    name: cpp-hello-world
+    source:
+      type: git
+      location: 'https://github.com/che-samples/cpp-hello-world'
+components:
+-
+  type: chePlugin
+  id: che-incubator/cpptools/latest
+  alias: cpp-plugin
+  preferences:
+    clangd.path: /usr/bin/clangd
+    cdt.clangd.binaries.enable: false
+-
+  type: dockerimage
+  alias: cpp-dev
+  image: quay.io/eclipse/che-cpp7-centos:nightly
```

Comment:
switch to use che-cpp-rhel7

Reply:
is this image available publicly?

Reply:
It will be available once I merge the PR